samtranslator/model/api/api_generator.py

```diff
@@ -93,6 +96,10 @@ def _construct_rest_api(self):
             raise InvalidResourceException(self.logical_id,
                                            "Specify either 'DefinitionUri' or 'DefinitionBody' property and not both")
 
+        if self.open_api_version and self.open_api_version not in OpenApiVersionsSupported:
```

Comment:
So I think people often use the patch version for OpenApi as well (ie '3.0.0' or '3.0.1'); I think we just want to verify that they're using some version of OpenApi 3 and not care about the minor/patch versions.

Reply:
Do you recommend adding a regex here?

Reply:
That would work